app/models/bulk_download.rb

```diff
@@ -158,16 +159,24 @@ def kickoff_ecs_task(
     end
   end
 
-  def get_param_value(key)
+  def get_param_field(key, field)
     # If params is nil, try to set it from params_json
     if params.nil? && params_json.present?
       self.params = JSON.parse(params_json)
```

Comment:
what's the point of `params`? is it only to avoid the cost of parsing the JSON on each get?

Reply:
Yeah that's right.